casanovo/denovo/dataloaders.py

```
@@ -342,6 +355,288 @@ def _make_dataset(
 
         return dataset
 
+    def _dia_to_dataframe(self, paths, annotated):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_dia_to_dataframe`

(ANN202)